lib/elixir/unicode/unicode.ex

```diff
@@ -482,4 +496,50 @@ defmodule String.Normalizer do
   end
 
   defp combining_class(_), do: 0
+
+  def nfd_to_nfc("", acc), do: acc
```

Comment:
Good catch! The irony is that I added syntax support for distinguishing private functions from public ones in Sublime Text, but I never bothered to utilize that distinction in my color scheme. :laughing: